### PR TITLE
docs: add explain analyze info for TableFullScan on TiFlash, and update related sample case

### PR DESCRIPTION
**Pull request by hongyunyan**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

1. add explain analyze info for `TableFullScan` on TiFlash, 
2. update related sample case for `Explain Analyze` with `TableFullScan` on TiFlash

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by breezewish** [comment]
Let's hold this PR until TiDB PR got merged. There may be naming changes before TiDB PR is merged.

File: sql-statements/sql-statement-explain-analyze.md
Comment (qiancai) [inline sql-statements/sql-statement-explain-analyze.md:229]:
```suggestion
### TableFullScan (TiFlash)
```

File: sql-statements/sql-statement-explain-analyze.md
Comment (qiancai) [inline sql-statements/sql-statement-explain-analyze.md:244]:
```suggestion
    - `total_scanned_packs`: 扫表过程中 dmfiles 中被扫描过的 pack 的数量
    - `total_skipped_packs`: 扫表过程中 dmfiles 中被跳过的 pack 的数量
    - `total_scanned_rows`: 扫表过程中 dmfiles 中被扫描过的 row 的数量
    - `total_skipped_rows`: 扫表过程中 dmfiles 中被跳过的 row 的数量 
    - `total_rough_set_index_load_time`: 扫表过程中相关 dmfiles 加载粗糙索引的总耗时
    - `total_read_time`: 扫表过程中读取 dmfiles 内数据的总耗时
+ `total_create_snapshot_time`: 扫表过程中创建快照的总耗时
```

File: sql-statements/sql-statement-explain-analyze.md
Comment (qiancai) [inline sql-statements/sql-statement-explain-analyze.md:244]:
```suggestion
+ dmfile：扫表过程中跟 dmfile 相关的执行信息
    - `total_scanned_packs`：扫表过程中 dmfiles 中被扫描过的 pack 的数量
    - `total_skipped_packs`：扫表过程中 dmfiles 中被跳过的 pack 的数量
    - `total_scanned_rows`：扫表过程中 dmfiles 中被扫描过的 row 的数量
    - `total_skipped_rows`：扫表过程中 dmfiles 中被跳过的 row 的数量 
    - `total_rough_set_index_load_time`：扫表过程中相关 dmfiles 加载粗糙索引的总耗时
    - `total_read_time`：扫表过程中读取 dmfiles 内数据的总耗时
+ `total_create_snapshot_time`：扫表过程中创建快照的总耗时
```

File: sql-statements/sql-statement-explain-analyze.md
Comment (qiancai) [inline sql-statements/sql-statement-explain-analyze.md:237]:
```suggestion
+ `dmfile`：扫表过程中与 dmfile 相关的执行信息
```

File: sql-statements/sql-statement-explain-analyze.md
Comment (breezewish) [inline sql-statements/sql-statement-explain-analyze.md:243]: Some users want to have a deeper understanding about our detailed information. I enriched a bit so that it could be more understandable for these advanced users:

```suggestion
+ `dtfile`：扫表过程中与 DTFile 相关的信息；这基本对应了 Stable 层数据的读取情况
    - `total_scanned_packs`：DTFile 内累计读取的 Pack 的数量
    - `total_skipped_packs`：DTFile 内累计跳过的 Pack 的数量
    - `total_scanned_rows`：DTFile 内累计读取的行数（若存在多版本更新或删除，则每个版本独立计数）
    - `total_skipped_rows`：DTFile 内累计跳过的行数
    - `total_rs_index_load_time`：读取 DTFile 粗糙索引的累计耗时
    - `total_read_time`：读取 DTFile 数据的累计耗时
```

Comment (breezewish) [reply sql-statements/sql-statement-explain-analyze.md:243]: @hongyunyan I also have a question, for "skipped packs", how is it counted? For example, may be we have range clips and RS masks, does it only count for RS masks? (i.e. skip == skip by RS index)

**Review by breezewish — APPROVED** [review]
The rest LGTM

File: sql-statements/sql-statement-explain-analyze.md
Comment (qiancai) [inline sql-statements/sql-statement-explain-analyze.md:237]:
```suggestion
+ `dtfile`：扫表过程中与 DTFile （即 DeltaTree File）相关的信息；这基本对应了 Stable 层数据的读取情况
```

File: sql-statements/sql-statement-explain-analyze.md
Comment (qiancai) [inline sql-statements/sql-statement-explain-analyze.md:240]: 多版本是指什么的多版本呢

Comment (hongyunyan) [reply sql-statements/sql-statement-explain-analyze.md:240]: 是指 MVCC 的多版本，比如 对同一行的信息进行了 update，就会有这一行对应的多版本信息。这边想表达的是，这些多个版本的行数都会参与整体的统计。

File: sql-statements/sql-statement-explain-analyze.md
Comment (qiancai) [inline sql-statements/sql-statement-explain-analyze.md:237]:
```suggestion
+ `dtfile`：扫表过程中与 DTFile （即 DeltaTree File）相关的信息；这基本反映了 Stable 层数据的读取情况
```

File: sql-statements/sql-statement-explain-analyze.md
Comment (qiancai) [inline sql-statements/sql-statement-explain-analyze.md:240]:
```suggestion
    - `total_scanned_rows`：DTFile 内累计读取的行数；若存在 MVCC 多版本更新或删除，则每个版本独立计数
```

File: sql-statements/sql-statement-explain-analyze.md
Comment (joey-yez) [inline sql-statements/sql-statement-explain-analyze.md:229]: 有的标题包含“执行信息”，有的没有包含，是否需要统一 @qiancai

Comment (qiancai) [reply sql-statements/sql-statement-explain-analyze.md:229]: 因为这里的二级标题里已经写了 “算子执行信息介绍”，如果每个三级标题后面都再写一个"执行信息“感觉比较重复，所以看起来都各三级标题里只包含算子名称比较统一和直接

File: sql-statements/sql-statement-explain-analyze.md
Comment (qiancai) [inline sql-statements/sql-statement-explain-analyze.md:237]: 请教下，什么是“Stable 层”

Comment (qiancai) [inline sql-statements/sql-statement-explain-analyze.md:237]:
```suggestion
+ `dtfile`：扫表过程中与 DTFile （即 DeltaTree File）相关的信息；这基本反映了 TiFlash 在 Stable 层数据的读取情况
```

File: sql-statements/sql-statement-explain-analyze.md
Comment (ran-huang) [inline sql-statements/sql-statement-explain-analyze.md:234]: 这个太长了，网页上查看的时候需要左右滑动才能看完整，而且滑动的时候滚动条会把文字挡住😂 可读性不太好。
image


如果不影响实际含义，能否手动 wrap 下，比如写成类似这种形式：

```suggestion
tiflash_scan:{
  dtfile:{
    total_scanned_packs:2, 
    total_skipped_packs:1, 
    total_scanned_rows:16000, 
    total_skipped_rows:8192, 
    total_rough_set_index_load_time: 2ms, 
    total_read_time: 20ms
  }, 
  total_create_snapshot_time: 1ms
}
```

File: sql-statements/sql-statement-explain-analyze.md
Comment (ran-huang) [inline sql-statements/sql-statement-explain-analyze.md:237]:
```suggestion
+ `dtfile`：扫表过程中与 DTFile （即 DeltaTree File）相关的信息；这基本反映了 TiFlash 在 Stable 层数据的读取情况。
```

File: sql-statements/sql-statement-explain-analyze.md
Comment (ran-huang) [inline sql-statements/sql-statement-explain-analyze.md:238]:
```suggestion
    - `total_scanned_packs`：DTFile 内累计读取的 Pack 的数量；Pack 是 TiFlash DTFile 读取的最小粒度，默认情况下每 8192 行构成一个 Pack。
```

File: sql-statements/sql-statement-explain-analyze.md
Comment (ran-huang) [inline sql-statements/sql-statement-explain-analyze.md:244]:
```suggestion
    - `total_skipped_packs`：DTFile 内累计跳过的 Pack 的数量；Pack 会由于 `WHERE` 条件命中粗糙索引或主键范围过滤而被跳过。
    - `total_scanned_rows`：DTFile 内累计读取的行数；若存在 MVCC 带来的多版本更新或删除，则每个版本独立计数。
    - `total_skipped_rows`：DTFile 内累计跳过的行数。
    - `total_rs_index_load_time`：读取 DTFile 粗糙索引的累计耗时。
    - `total_read_time`：读取 DTFile 数据的累计耗时。
+ `total_create_snapshot_time`：扫表过程中创建快照的总耗时。
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

File: sql-statements/sql-statement-explain-analyze.md
Comment (breezewish) [inline sql-statements/sql-statement-explain-analyze.md:239]:
```suggestion
    total_scanned_packs: 2, 
    total_skipped_packs: 1, 
    total_scanned_rows: 16000, 
    total_skipped_rows: 8192, 
```

Comment (breezewish) [inline sql-statements/sql-statement-explain-analyze.md:239]:
```suggestion
tiflash_scan: {
  dtfile: {
    total_scanned_packs: 2, 
    total_skipped_packs: 1, 
    total_scanned_rows: 16000, 
    total_skipped_rows: 8192, 
```

**Review by ran-huang — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**
